pkgs/servers/home-assistant/custom-components/mypyllant/package.nix

```diff
@@ -28,6 +29,14 @@ buildHomeAssistantComponent rec {
     hash = "sha256-6T8SGAP2535VqZmvSeITpMIa0SBJhnWsOKM1Y66WhHE=";
   };
 
+  patches = [
+    (fetchpatch2 {
+      name = "fix-tests-pr-394.patch";
+      url = "https://github.com/signalkraft/mypyllant-component/pull/394.patch";
+      hash = "sha256-V/0oBCuPjU6oVvtq+Zcw+lUnrtcgwbEFqM9NK7wPgsQ=";
+    })
```

Comment:
PR refs are volatile and as the PR is not merged yet, please instead drop the patch into nixpkgs.

Reply:
We only need the [‎tests/test_services.py‎](https://github.com/signalkraft/mypyllant-component/pull/394/changes#diff-caee650c3003842e9277d696a063ab724d42606e64cbf44cf1deb9d1d67d1d98) hunks. We don't care about pyproject/uv version pins.